### DISCUSSION TITLE
Is it possible to have one of the many particles follow the mouse cursor?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [matteobruni/tsparticles#5252] [General]
Is it possible to have one of the many particles follow the mouse cursor?

**Comment by matteobruni** [comment]
> Is it possible to have one of the many particles follow the mouse cursor?

Not natively, you need to create a custom external interactivity plugin

**Reply by ivanjeremic** [reply]
Has anyone ever attempted something similar, any code out there I can take a look at?

**Reply by matteobruni** [reply]
https://github.com/tsparticles/tsparticles/tree/main/interactions/external

These are the existing plugins, it’s not necessary to create a package, you can have that code in your project only.
